### PR TITLE
[NUI][ATSPI]Move responsibility of calculation CHECKED atspi state from SelectButton to Button class

### PR DESCRIPTION
**Pull request by grzelewski**
### Description of Change ###
This change will fix reading problem with Toggle button.

### API Changes ###
None

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Button.cs [hunk 1]
```diff
@@ -190,6 +190,7 @@ public Button(ButtonStyle buttonStyle) : base(buttonStyle)
         protected override AccessibilityStates AccessibilityCalculateStates()
         {
             var states = base.AccessibilityCalculateStates();
+            states.Set(AccessibilityState.Checked, this.IsSelected);
```
Comment (lol-github) [inline src/Tizen.NUI.Components/Controls/Button.cs:193]: As "selected" state is supported in more general class (Button) it looks OK to move here the mapping to ATSPI for that state.

**Review by lol-github — APPROVED** [review]
As "selected" state is supported in more general class (Button) it looks OK to move there the mapping to ATSPi for that state.

**Review by lol-github — CHANGES_REQUESTED** [review]
please update PR title to give the real intention of this PR, like "[NUI][ATSPI]Move responsibility of calculation CHECKED atspi state from SelectButton to Button class"

**Comment by grzelewski** [comment]
> 
> 
> please update PR title to give the real intention of this PR, like "[NUI][ATSPI]Move responsibility of calculation CHECKED atspi state from SelectButton to Button class"

done

File: src/Tizen.NUI.Components/Controls/Button.cs (hunk 1 above)
Comment (grzelewski) [reply src/Tizen.NUI.Components/Controls/Button.cs:193]: Done.

**Review by lol-github — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]

**Pull request merged**